packages/overlay/package.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "@swc-uxp-wrappers/overlay",
+    "version": "1.0.0",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "UXP wrapper of the @spectrum-web-components/overlay component",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://git.corp.adobe.com/torq/swc-uxp-wrappers.git",
+        "directory": "packages/overlay"
+    },
+    "author": "Adobe Inc",
+    "main": "src/index.js",
+    "module": "src/index.js",
+    "type": "module",
+    "exports": {
+        ".": "./src/index.js",
+        "./package.json": "./package.json",
+        "./src/AbstractOverlay.js": "./src/AbstractOverlay.js",
+        "./src/Overlay.js":  "./src/Overlay.js",
+        "./src/OverlayDialog.js": "./src/OverlayDialog.js",
+        "./src/OverlayNoPopover.js": "./src/OverlayNoPopover.js",
+        "./src/OverlayPopover.js": "./src/OverlayPopover.js",
+        "./src/OverlayStack.js": "./src/OverlayStack.js",
+        "./src/OverlayTrigger.js":  "./src/OverlayTrigger.js",
+        "./src/PlacementController.js": "./src/PlacementController.js",
+        "./src/VirtualTrigger.js": "./src/VirtualTrigger.js",
+        "./src/fullSizePlugin.js":  "./src/fullSizePlugin.js",
+        "./src/index.js":  "./src/index.js",
+        "./src/loader.js": "./src/loader.js",
+        "./src/overlay-events.js":  "./src/overlay-events.js",
+        "./src/overlay-timer.js":  "./src/overlay-timer.js",
+        "./src/overlay-trigger.css.js": "./src/overlay-trigger.css.js",
+        "./src/overlay-types.js": "./src/overlay-types.js",
+        "./src/overlay.css.js": "./src/overlay.css.js",
+        "./src/topLayerOverTransforms.js": "./src/topLayerOverTransforms.js",
+        "./active-overlay.js":  "./active-overlay.js",
+        "./overlay-trigger.js": "./overlay-trigger.js",
+        "./sync/overlay-trigger.js":  "./overlay-trigger-sync.js",
```

Comment:
we dont have any folder sync and file overlay-trigger.js?

Reply:
We don't have it but it is there in swc folder structure so I just re-exported to the file I have created inside our package.